src/main/java/org/jenkinsci/plugins/xunit/XUnitChecksPublisher.java

```diff
@@ -173,6 +169,10 @@ private String extractChecksTitle() {
             return "There were test failures";
         }
 
+        if (summary.getTotalCount() == summary.getSkipCount()) {
```

Comment:
please split conditions
summary.getTotalCount() == 0 -> return "No test result found"
summary.getTotalCount() == summary.getSkipCount() -> return "All tests were skipped"

Reply:
@nfalco79 The conditions were already splited.  The first condition in the method is the `summary.getTotalCount() == 0` and the last condition is the `summary.getTotalCount() == summary.getSkipCount()`.
I only adjusted the message as you suggested.